src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -6,49 +6,82 @@ redirect_from:
 
-Auth Tokens are passed using an auth header, and are used to authenticate as a user account with the API. The [Public Integration](#public-integrations) requires an OAuth flow for tokens. The [Internal Integration](#internal-integrations) automatically generates tokens after installation. For more information, see the [full documentation on Authentication](/api/auth/).
+Webhooks allow your service to get requests about specific resources, depending on your selection. For more information, check out the [full documentation on Webhooks](/product/integrations/integration-platform/webhooks/).
 
-### Integration Webhooks
+## Using Auth Tokens
 
-Webhooks allow your service to get requests about specific resources, depending on your selection. For more information, see the [full documentation on Webhooks](/product/integrations/integration-platform/webhooks/).
+Auth tokens are passed using an auth header, and are used to authenticate as a user account with the API. [Public integrations](#public-integrations) require an OAuth flow for tokens. [Internal integrations](#internal-integrations) automatically generate tokens after installation. For more information, check out the [full documentation on Authentication](/api/auth/).
 
 ## Public Integrations
 
-Sentry built public integrations for the 'general public' of Sentry users. Public integrations start in an unpublished state for development purposes and can later be submitted for approval to publish. For more information, see the [section on Publishing](#published-state).
+Public integrations are for the "general public" of Sentry users. These start in an unpublished state for development purposes and can later be submitted for approval to publish. For more information, check out the [section on Publishing](#published-state).
 
 The code examples in the sections below demonstrate a potential use-case that involves a Flask app receiving new issue webhooks from Sentry, calling the Sentry API for more data about the issue, and pushing it to Pushover as a generator of desktop/mobile notifications.
 
 ### Installation
 
-Users will have the option to install your integrations on the Integrations Page in Sentry. If your integration is still in an unpublished state, the only Sentry organization that will be able to install that integration will be the organization that created the integration. Clicking "Install" will allow users to see a description of your integration and the permissions that will be granted should the user choose to install.
+Users will have the option to install your integrations on the "Integrations" page in [sentry.io](https://sentry.io). If your integration is still in an unpublished state, the only Sentry organization that will be able to install it will be the organization that created the integration. Clicking the name of the integration will allow users to see a description of your integration and the permissions that will be granted if the user chooses to install.
```

Comment:
```suggestion
Users will have the option to install your integrations on the **Integrations** page in [sentry.io](https://sentry.io). If your integration is still in an unpublished state, the only Sentry organization that will be able to install it will be the organization that created the integration. Clicking the name of the integration will allow users to see a description of your integration and the permissions that will be granted if the user chooses to install.
```